src/mpyl/steps/deploy/k8s/chart.py

```diff
@@ -879,6 +897,7 @@ def _to_service_components_chart(builder):
         f"{builder.project.name}-ingress-{i}-http": route
         for i, route in enumerate(builder.to_ingress_routes(https=False))
     }
+    ingress = {"ingress": builder.to_ingress()}
```

Comment:
```suggestion
    ingress = {"ingress-routes": builder.to_ingress()}
```

So that it matches [the metadata name](https://github.com/Vandebron/gh-mpyl/pull/103/files?diff=split&w=1#diff-f2e93ed09e3ea127c21228d5487fed442fe1b3364117f6a7ec4705cd660a72a0R516).